samples/System.Numerics.Tensors.Samples/TensorSamples/MutateSparse.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace TensorSamples
+{
+    class MutateSparse
+    {
+        public static void RunSample()
+        {
+            Console.WriteLine($"*** {nameof(MutateSparse)} ***");
+            CompressedSparseTensor<double> tensor = new CompressedSparseTensor<double>(new[] { 3, 5, 4 }, 10);
+
+            tensor[0, 2, 0] = 1.6;
+            tensor[1, 0, 3] = 5.3;
+            tensor[1, 1, 0] = Math.PI;
+            tensor[2, 3, 3] = 867.5309;
+            tensor[2, 4, 0] = 19.4231;
+            tensor[2, 4, 3] = 10000.1;
+
+            Console.WriteLine("Original tensor:");
+            Console.WriteLine(tensor.GetArrayString());
+
+            ScalarPowerSparse(tensor, 3);
+
+            Console.WriteLine("Cubed tensor:");
+            Console.WriteLine(tensor.GetArrayString());
+        }
+        
+        // The following represent what .NET bindings would look like for an native library that wishes
+        // to deal take a sparse tensor and mutate its values        
+
+        public static unsafe double ScalarPowerSparse(CompressedSparseTensor<double> tensor, int exponent)
+        {
+            fixed (double* valuesPtr = &tensor.Values.Span.DangerousGetPinnableReference())
```

Comment:
These feel a bit clunky, not to mention the lack of intellisense.  Is this really the best way to interop with span?

Reply:
/cc @KrzysztofCwalina @ahsonkhan @stephentoub

Reply:
@VSadov, long term we will get C# fixed statement to generate this code.

Reply:
Will try enabling `fixed(double* valuesPtr = tensor.Values.Span)` in 7.3

Note that you can also do `fixed(double* valuesPtr = &tensor.Values.Span[0])`

Reply:
> Note that you can also do `fixed(double* valuesPtr = &tensor.Values.Span[0])` 

If the span is empty, we can't do that.